source/clear-linux/get-started/virtual-machine-install/vmw-player.rst

```diff
@@ -66,7 +66,7 @@ Download the latest |CL| installer ISO
 **************************************
 
 Get the latest |CL| installer ISO image from the `image`_ repository.
```

Comment:
Let's point them to https://clearlinux.org/downloads instead now that it exists

Reply:
good call, updated with latest commit.